walkeeper/src/safekeeper.rs

```diff
@@ -470,14 +470,12 @@ struct SafeKeeperMetrics {
 }
 
 impl SafeKeeperMetrics {
-    fn new(tenant_id: ZTenantId, timeline_id: ZTimelineId, commit_lsn: Lsn) -> Self {
```

Comment:
Why that was removed?

Reply:
It was used in `SafeKeeper::new` and the logic was to set metric to the value read from `state.commit_lsn`. It's unnecessary, because we still set metric value on every commit_lsn update.